base/src/main/java/uk/ac/ebi/atlas/bioentity/properties/BioEntityCardModelFactory.java

```diff
@@ -158,6 +148,10 @@ private String getFirstValueOfProperty(BioentityPropertyName propertyType,
                                 )
                 ).collect(Collectors.toList());
 
+        // Sort by relevance in descending order
+        links.sort((PropertyLink p1, PropertyLink p2) -> p2.getRelevance() - p1.getRelevance());
+
+        return links;
```

Comment:
Java streams have a [`.sorted`](https://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.html#sorted-java.util.Comparator-) method that you can chain, so this could be:
```
return bioEntityPropertyService.mapToLinkText(propertyName, propertyValues).entrySet().stream()
    ...
    .sorted((p1, p2) -> p2.getRelevance() - p1.getRelevance())
    .collect(toList())
```

The comparator can further be simplified to `.sorted(comparing(PropertyLink::getRelevance))` but check docs first.